tests/load-tests/run-max-concurrency.sh

```diff
@@ -73,10 +73,9 @@ else
         index=$(printf "%04d" "$t")
         cp -vf "$output_dir/load-tests.json" "$output_dir/load-tests.max-concurrency.$index.json"
         cp -vf "$output_dir/load-tests.log" "$output_dir/load-tests.max-concurrency.$index.log"
-        pipelineRunThresholdExceeded=$(jq -rc ".runPipelineSucceededTimeMax > $threshold" "$output_dir/load-tests.json")
-        pipelineRunKPI=$(jq -rc ".runPipelineSucceededTimeMax" "$output_dir/load-tests.json")
-        if [ "$pipelineRunThresholdExceeded" = "true" ]; then
-            echo "The maximal time a pipeline run took to succeed (${pipelineRunKPI}s) has exceeded a threshold of ${threshold}s with $t threads."
+        workloadKPI=$(jq '.createApplicationsTimeAvg + .createCDQsTimeAvg + .createComponentsTimeAvg + .integrationTestsRunPipelineSucceededTimeAvg + .runPipelineSucceededTimeAvg + .deploymentSucceededTimeAvg' "$output_dir/load-tests.json")
+        if [ "$workloadKPI" -gt "$threshold" ]; then
+            echo "The maximal time a workload took to succeed (${workloadKPI}s) has exceeded a threshold of ${threshold}s with $t threads."
```

Comment:
The KPI here is not a maximal time any more, now... you are accumulating averages, so the actual KPI here is average time for workload to succeed, not maximal.

Reply:
If you want to capture the maximal time to succeed, then the load test itself should sum the actual values of the above times for each user individually as it executes and then report the maximum to the output JSON at the end of execution.

Reply:
Yeah, I simplified it to just summing averages. I think it is good enough. Mine next step here is to employ a bit of math and using result from previous value determine KPI where we crossed 300 seconds threshold exactly (in average).

Reply:
I see, then the message should change the `maximal time` to `average time`, at least :)
Also, I expect thant with this change the number of max concurrency reached would go higher with using avg instead of max, so the concurrency steps would need to be revised in CI to reflect that. Let's run the `max-concurrency-*` jobs to see how it goes.

Reply:
Aaah, you are right, thanks.